src/components/body/index.js

```diff
@@ -36,8 +37,10 @@ function getElementComponent(type) {
     case 'embeddedCode':
     case 'embeddedcode':
       return null
+    case 'small-image':
```

Comment:
add `small-image` in `src/constants/prop-types.js` ?

Reply:
forgot about it, thx